app/test/block_size_test.go

```diff
@@ -309,5 +309,5 @@ func queryWithOutProof(clientCtx client.Context, hashHexStr string) (*rpctypes.R
 		return nil, err
 	}
 
-	return node.Tx(context.Background(), hash, false)
+	return node.Tx(context.Background(), hash, prove)
```

Comment:
this change requires that we include a tx inclusion proof when querying for a tx